qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -371,31 +390,50 @@ def new_spin_echo_sequence(duration=None, pre_post_rotation=False, **kwargs):
 def new_carr_purcell_sequence(
     duration=None, number_of_offsets=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates Carr-Purcell Sequence.
+    r"""
+    Creates the Carr-Purcell sequence.
 
     Parameters
     ---------
     duration : float, optional
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     number_of_offsets : int, optional
-        Number of offsets. Defaults to None
+        Number of offsets :math:`n`. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Carr-Purcell sequence
+    DynamicDecouplingSequence
+        The Carr-Purcell sequence.
+
+    See Also
+    --------
+    new_cpmg_sequence
+
+    Notes
+    -----
+    The Carr-Purcell sequence [#]_ is parameterized by the number of offsets :math:`n`
+    and duration :math:`\tau`. The sequence is made up of a set of :math:`X_{\pi}`
+    operations applied at
+
+    .. math::
+        t_i = \frac{\tau}{n} \left( \frac{1}{2} +i - 1\right) \;,
```

Comment:
optional:
```suggestion
        t_i = \frac{\tau}{n} \left( i - \frac{1}{2}\right) \;,
```

(similarly for CPMG)

Reply:
yeah, this looks better. Fixed.